CodePushPackage.m

```diff
@@ -259,9 +266,9 @@ + (void)downloadPackage:(NSDictionary *)updatePackage
                         return;
                     }
                     
-                    [CodePushPackage copyEntriesInFolder:currentPackageFolderPath
-                                              destFolder:newPackageFolderPath
-                                                   error:&error];
+                    [[NSFileManager defaultManager] copyItemAtPath:currentPackageFolderPath
+                                                            toPath:newPackageFolderPath
```

Comment:
It creates the destination folder if it doesnt exist, and fails if it already does